drivers/RP2040/grbl/config.h

```diff
@@ -74,7 +74,7 @@
 // immediately forces a feed hold and then safely de-energizes the machine. Resuming is blocked until
 // the safety door is re-engaged. When it is, Grbl will re-energize the machine and then resume on the
 // previous tool path, as if nothing happened.
-// #define ENABLE_SAFETY_DOOR_INPUT_PIN // Default disabled. Uncomment to enable.
+#define ENABLE_SAFETY_DOOR_INPUT_PIN // Default disabled. Uncomment to enable.
```

Comment:
This change looks like a leftover from testing. If the default was meant to be changed though, then the comment is now incorrect.

Reply:
@kdomanski Use the version from the new repo, I have merged the relevant changes there already. The driver will no longer be maintained here and I will soon add a readme stating that.